authorization/src/main/java/org/apache/atlas/authorize/simple/AtlasSimpleAuthorizer.java

```diff
@@ -350,6 +350,11 @@ public void scrubSearchResults(AtlasSearchResultScrubRequest request) throws Atl
         }
     }
 
+    @Override
+    public void scrubSearchResults(AtlasSearchResultScrubRequest request, boolean isScrubAuditEnabled) throws AtlasAuthorizationException {
+        scrubSearchResults(request);
```

Comment:
variable `isScrubAuditEnabled` not used

Reply:
We will handle this in [this](https://linear.app/atlanproduct/issue/PLT-636/return-personapurposepolicy-details-in-the-scrubbed-assets) chnage